onnx/cpp2py_export.cc

```diff
@@ -114,6 +114,22 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
       .def_property_readonly("since_version", &OpSchema::since_version)
       .def_property_readonly("deprecated", &OpSchema::deprecated)
       .def_property_readonly("domain", &OpSchema::domain)
+      .def_property_readonly("function_opset_versions", &OpSchema::function_opset_versions)
+      .def_property_readonly(
+          "context_dependent_function_opset_versions", &OpSchema::context_dependent_function_opset_versions)
+      .def_property_readonly(
+          "all_function_opset_versions",
+          [](OpSchema* op) -> std::vector<int> {
+            std::vector<int> all_function_opset_versions = op->function_opset_versions();
+            std::vector<int> context_dependent_function_opset_versions =
+                op->context_dependent_function_opset_versions();
+            all_function_opset_versions.insert(
+                all_function_opset_versions.end(),
+                context_dependent_function_opset_versions.begin(),
+                context_dependent_function_opset_versions.end());
+            std::sort(all_function_opset_versions.begin(), all_function_opset_versions.end());
```

Comment:
Is there a possibility the result contains duplicates?

Reply:
good point, added code to remove duplications. op authors shall not define both type of functions with the same version. all_function_opset_versions is used to check whether there is any function so we generate doc properly.